exe/stripe-dev

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env ruby
+#
+# Fetches Stripe development environment variables from 1Password and starts
+# the Stripe CLI webhook listener.
+#
+# Usage: eval "$(bundle exec stripe-dev)"
+
+require "json"
+require "fileutils"
+
+LOG_FILE = "log/stripe.development.log"
+PID_FILE = "tmp/stripe.tunnel.pid"
+
+# Ensure directories exist
+FileUtils.mkdir_p("log")
+FileUtils.mkdir_p("tmp")
+
+# Kill any existing stripe tunnel process
+if File.exist?(PID_FILE)
+  old_pid = File.read(PID_FILE).strip.to_i
+  if old_pid > 0
+    Process.kill("TERM", old_pid) rescue nil
+  end
+  File.delete(PID_FILE)
+end
+
+# Fetch secrets from 1Password
+secrets_escaped = `kamal secrets fetch \
+  --adapter 1password \
+  --account basecamp \
+  --from "Deploy/Fizzy" \
+  "Development/STRIPE_SECRET_KEY" \
+  "Development/STRIPE_MONTHLY_V1_PRICE_ID" 2>/dev/null`
+
+secrets_json = secrets_escaped.gsub(/\\(.)/, '\1')
+secrets = JSON.parse(secrets_json)
+
+stripe_secret_key = secrets["Deploy/Fizzy/Development/STRIPE_SECRET_KEY"]
+stripe_price_id = secrets["Deploy/Fizzy/Development/STRIPE_MONTHLY_V1_PRICE_ID"]
```

Comment:
@jorgemanrubia curious if you looked at using `op run` for handling the secrets, rather than Kamal?

It ought to take care of some of these details for you, and would let you move the secret & env names into a file like `.stripe-env` rather than have them inside the script. Then you could exec both the `stripe listen` and `bin/dev` with the necessary secrets in place.